businfo/businfo.go

```diff
@@ -0,0 +1,28 @@
+package businfo
+
+import (
+	businfo "github.com/fahimimam/busbdChckr/businfo/availableBusInformation"
+	"github.com/fahimimam/busbdChckr/businfo/models"
+)
+
+func GetBusInfoV2(data businfo.RequestPld) ([]*models.NotificationPld, error) {
```

Comment:
No need to call it V2 since the V1 was just a POC.

Reply:
Okey Sure thing